Tests/AK/TestStringUtils.cpp

```diff
@@ -321,6 +321,10 @@ TEST_CASE(contains)
     EXPECT(!AK::StringUtils::contains("", test_string, CaseSensitivity::CaseInsensitive));
     EXPECT(!AK::StringUtils::contains(test_string, "L", CaseSensitivity::CaseSensitive));
     EXPECT(!AK::StringUtils::contains(test_string, "L", CaseSensitivity::CaseInsensitive));
+
+    String command_palette_bug_string = "Go Go Back";
+    EXPECT(AK::StringUtils::contains(command_palette_bug_string, "Go Back", AK::CaseSensitivity::CaseSensitive));
+    EXPECT(AK::StringUtils::contains(command_palette_bug_string, "Go Back", AK::CaseSensitivity::CaseInsensitive));
```

Comment:
should actually do a case insensitive lookup here. :)

Reply:
That's a very good point. Fixed!